packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -1007,4 +1007,11 @@ whisperkit-cli transcribe --audio-path /path/to/audio.mp3
 # Or use your preferred model variant
 whisperkit-cli transcribe --model "large-v3" --model-prefix "distil" --audio-path /path/to/audio.mp3 --verbose`,
 ];
+
+export const threedtopia_xl = (): string[] => [
```

Comment:
Please put an actual model snippet here instead of a URL.

Reply:
Updated with actual snippet together with URL to detailed instructions!